.travis.yml

```diff
@@ -1,3 +1,10 @@
+branches:
+  only:
+    - master
+    - major
+    - minor
+    - bugfix
```

Comment:
why don't we want to run on any other branch? I think this should be a different PR if there's a reason for it (the table looks good to me now.)